backend/test/test_collections.py

```diff
@@ -1,3 +1,4 @@
+from pickle import PickleBuffer
```

Comment:
typo / extra line added?

Reply:
Oops yeah, I think aggressive LSP imported it by mistake